src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true'">
-    <TargetManifestFiles>$(TargetManifestFiles);$(MSBuildThisFileDirectory)manifest.win7-x64.xml;$(MSBuildThisFileDirectory)manifest.win7-x86.xml;$(MSBuildThisFileDirectory)manifest.osx-x64.xml;$(MSBuildThisFileDirectory)manifest.linux-x64.xml</TargetManifestFiles>
+    <TargetManifestFiles>$(TargetManifestFiles);$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-win7-x64.xml;$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-win7-x86.xml;$aspnetcore-store-__MANIFEST_VERSION__-osx-x64.xml;$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-linux-x64.xml</TargetManifestFiles>
```

Comment:
It looks like you just add this to the `TargetManifestFileList` item group.

```xml
  <ItemGroup Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true'">
    <TargetManifestFileList Include="$(MSBuildThisFileDirectory)aspnetcore-store-*.xml" />
  </ItemGroup >
```

cref https://github.com/dotnet/sdk/blob/43090565482d9a10481ff2298fb168fc819fa430/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets#L33